src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -61,11 +62,34 @@ public ExamResource(UserService userService, CourseService courseService, ExamSe
         this.authCheckService = authCheckService;
     }
 
+    /**
+     * POST /courses/{courseId}/exams/{examId} : Get a exam by id
+     *
+     * @param courseId the id of the course to which the exam belongs
+     * @param examId   the id of the exam to get
+     * @return ResponseEntity with exam in the body if it can be found, otherwise not found
+     */
+    @GetMapping("/courses/{courseId}/exams/{examId}")
+    @PreAuthorize("hasAnyRole('USER', 'TA', 'INSTRUCTOR', 'ADMIN')")
```

Comment:
I think we should only allow this for instructors and admins for now (as this is part of the exam management)